### PR TITLE
Tests

### LINKED ISSUE #3233: More Tests!
Here are the goals that we'd like hit for the next two weeks with testing and notes from @tmashuang and I's talk:

- metamask-controller: 85-75 (curent: 42)
- network:
  bare min is to get the test properly stubbed
- blacklist:
  bare min is to get the test properly stubbed

### PR DESCRIPTION
**Pull request by tmashuang**
Fixes #3233.
https://coveralls.io/builds/17684713

### CODE REVIEW

File: test/unit/app/typed-message-manager.spec.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+describe('', function () {
+
+  let typedMessageManager, msgParams, msgParams2, typedMsgs, messages, msgId, numberMsgId
+
+  const address = '0xc42edfcc21ed14dda456aa0756c153f7985d8813'
+
+  beforeEach('', function () {
+    typedMessageManager = new TypedMessageManager()
+
+    msgParams = {
```
Comment (jennypollack) [inline test/unit/app/typed-message-manager.spec.js]: Typed message format [EIP-712](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md)

`"data": "{\"types\":{\"foo\":[]},\"message\":{},\"primaryType\":\"foo\",\"domain\":{}}",`

**Comment by bdresser** [comment]
@tmashuang is this ready or still WIP?

**Comment by tmashuang** [comment]
This is ready to be merged, will update when the new `signTypedData` gets updated in a separate PR.

**Comment by bdresser** [comment]
@tmashuang sign typed data is in, this ready to update?

**Comment by tmashuang** [comment]
Closing separate out this in multiple PRs.

**Pull request closed without merging**